tools/doms/doms_reader.py

```diff
@@ -0,0 +1,133 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from netCDF4 import Dataset
+import sys
+import datetime
+import csv
+from collections import OrderedDict
+

```

Comment:
General: it's best to setup logging even if you're only going to use it once or twice. It's pretty easy to do, just add
```
import logging

LOGGER = logging.getLogger("doms_reader")
```
At the top of the module. Then inside of your `if __name__ == '__main__':` add
```
logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s',
                    level=logging.INFO,
                    datefmt='%Y-%m-%d %H:%M:%S')
```

Then anywhere you want to log a message you can use `LOGGER.info("some message")`, etc...